src/main.cpp

```diff
@@ -452,10 +457,12 @@ void contactBackend()
     String apikey_restored = readStringFromEEPROM(EEPROM_ADDRESS_APIKEY);
     String temp = String(celsius);
     String relais1 = String((int)ac1.getCurrentValue());
-    String url = "https://bricks.bierbot.com/api/iot/v1?apikey=" + apikey_restored + "&type=" + "sonoff_th_origin" + "&brand=" + "bierbot" + "&version=" + global_version + "&s_number_temp_0=" + temp + "&a_bool_epower_0=" + relais1 + "&chipid=" + chipid + "&s_number_temp_id_0=" + sensor_id;
```

Comment:
Is there a way to ensure that `a_bool_epower_1` is only added for elite devices (i.e. the device id or type id or something). If not, we need to modify the build script to support building two releases in the future (Elite and non-elite).